components/esp_websocket_client/examples/pytest_websocket.py

```diff
@@ -65,19 +63,19 @@ def __exit__(self, exc_type, exc_value, traceback):
         if self.thread.is_alive():
             Utility.console_log('Thread cannot be joined', 'orange')
 
-
+@pytest.mark.skip()
 def test_echo(dut):
     dut.expect('WEBSOCKET_EVENT_CONNECTED')
     for i in range(0, 5):
-        dut.expect(re.compile(r'Received=hello (\d)'), timeout=30)
+        dut.expect('Received=hello')
```

Comment:
Does not the `expect()` API support regex anymore? Why have you removed the timeouts?